continuous_integration/recipe/meta.yaml

```diff
@@ -24,14 +24,17 @@ requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - maturin                                # [build_platform != target_platform]
     - libprotobuf                            # [build_platform != target_platform]
+    - zlib                                   # [build_platform != target_platform]
```

Comment:
Curious how `zlib` is being used during the build process

Reply:
Can try to dig up a link but seemed like the `linux-aarch64` builds seemed to fail if this wasn't included in the build requirements in addition to host